### ISSUE TITLE
Different colours in each quarter / per saltire?

### ISSUE THREAD

**Issue by karlwilcox** [drawshield/Drawshield-Code#48]
Is this valid? Per saltire gules or azure argent i.e. one colour per section
Can the same be said of quartered? Gyronny (of n)?

Fairly easy to parse but tricky to implement because (at the moment) we don't have separate objects for the various parts, so a lot of drawing would be required.

**Comment by karlwilcox** [comment]
Found some relevant correspondence:

I've just been having fun with your DrawShield, trying to figure out the blazon for a gyronny with more than two colors.
I managed to construct the shield I wanted (more or less) using this blazon:
"Quarterly 1st per bend azure and sable, 2nd per bend sinister sable and purpure, 3rd per bend sinister vert and sable, 4th per bend sable and gules"

But I think it would be more elegant to be able to describe same shield with the blazon:
"Gyronny azure, sable, purpure, sable, gules, sable, vert, sable"

Would it be possible/easy for DrawShield to support this expansion of gyronny?

Here's two links where I found discussion of gyronny with more than two colors:
https://oscar.sca.org/index.php?action=100&loi=4166
https://archive.org/details/treatiseonherald00wooduoft/page/86

Relevant quote:
"Gyronny is sometimes composed of more than two tinctures, thus a branch of the Milanese family of ORIGO bears: Gyronny, sable, argent, vert, sable, argent, vert, sable, vert."

Would love to hear your thoughts on this! - Luke
